test.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
@kristinh12 Can we be a bit more descriptive with this file name? `test.html` doesn't give that much context. Maybe `widget-no-description-test.html` or something like that.

Reply:
Ah I can probably just delete that, I wrote an actual test widget.spec.js.  The test.html was to try to duplicate the problem and have a visual of it working.

Reply:
I think these are super help though. Maybe we can add a folder to keep all of these? What do you think @sarahyablok ?

Reply:
We had an examples folder which got deleted cause it had bad examples. Perhaps for now we can delete this and file a ticket to create a new ticket for a new examples folder.

Reply:
Sounds good! I'm ok with that! 👍